reconcile/utils/oc_connection_parameters.py

```diff
@@ -105,6 +105,9 @@ class OCConnectionParameters:
 
     @staticmethod
     def _get_token_verify_server_url(secret: ClusterSecret, cluster: Cluster) -> str:
+        # Edge case, new cluster
+        if cluster.server_url == "":
```

Comment:
Please use `if not cluster.server_url:`

Reply:
Can do, but why?

Reply:
Because it's more [pythonic](https://letmegooglethat.com/?q=what+is+pythonic+code)